packages/wallet/api/src/marshal-contexts.js

```diff
@@ -197,6 +204,7 @@ export const makeExportContext = () => {
   };
 
   return harden({
+    saveOfferResultActions: makeSaver('offerResult', walletObjects.offerResult),
```

Comment:
The `Actions` part of this / these names is a little iffy. hm.

Reply:
Yeah. I opted for consistency with the others, but I don't understand why "Actions" is part of any of them. Cool to remove from all?

Reply:
I think so. @michaelfig ?

are the other 2 used? @samsiegart ?